src/main/java/command/ListCommand.java

```diff
@@ -0,0 +1,30 @@
+package command;
+
+import tasklist.TaskList;
+import utility.Storage;
+import utility.Ui;
+
+/**
+ * Represents command for list keyword
+ */
+public class ListCommand extends Command {
+
+    /**
+     * Instantiates a new list command
+     */
+    public ListCommand() {
+        super("list");
+    }
+
+    /**
+     * Executes the list command
```

Comment:
Same comment as above - this doesn't really explain what the list command actually does.